RFC-010-core-strategy/core-strategy.md

```diff
@@ -0,0 +1,65 @@
+# Executive Summary
+
+The Good Docs Project has grown organically since its creation, and has blossomed into a thriving community. Now it’s important to articulate where we are, how we can best support our users and community members, where we’d like to grow in future. 
+
+
+## Core Strategy Statement:  
+
+_The Good Docs Project educates and empowers people to create high quality documentation by providing them with resources, best practices and tools to enhance their documentation in open source and beyond._
+
+
+## What we do
+
+First and foremost, we create templates with supporting meta-documents that underpin the templates.
```

Comment:
```suggestion
First and foremost, we create templates with supporting meta-documents to facilitate template usage.
```
Might be more concise?

Reply:
Yes, I think your idea is better @kickoke - to "help use" rather than "underpin". However, I'm allergic to "usage" 😂. 
Leaving this one open to see if anyone can help with phrasing.  My brain is out to lunch.

Reply:
Resolving this comment. This sentence got fixed as a result of another comment and I can't remember why/why/how.